### PR TITLE
Feature/apvs 196/claim list and details

### PR DESCRIPTION
**Pull request by tswann**
- Added Claims List View
- Added Claim Detail View
- Pending addition of external app claim creation, a /refresh-claim route will generate a dummy record
- Re-factored the db-client file for reuse by various collections

### CODE REVIEW

**Comment by tmrowe** [comment]
I think a link from the index page to the new claims page would be handy.

**Comment by tswann** [comment]
@pwright08 @tmrowe - Hi guys can someone please review this update? I have uncommented the menu options for the gov template and added links for Eligibility and Claims lists.

File: alpha/internal-web/app/views/claim-details.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+{% block page_title %}
+{{ claim['reference'] }} {{ claim['claimant']['name'] }} Claim Details
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+    {% include "includes/phase_banner_alpha.html" %}
+
+    <div class='grid-row'>
```
Comment (stevenalexander) [inline alpha/internal-web/app/views/claim-details.html:12]: not using grid layout and this breaks column alignment

File: alpha/internal-web/app/views/claim-details.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <h1 class="heading-large">{{ claim['claimant']['name'] }}
+            <span class="heading-secondary">Claim Application</span>
+        </h1>
+
+        <div class="column-two-thirds">
```
Comment (stevenalexander) [inline alpha/internal-web/app/views/claim-details.html:17]: don't need this if not using grid

File: alpha/internal-web/app/views/claims-list.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+<main id="content" role="main">
+  {% include "includes/phase_banner_alpha.html" %}
+
+  <div class="grid-row">
+
+    <div class="column-two-thirds">
+      <h2 class="heading-medium">Claim Applications</h2>
+
+      <table id="claims-table" class="table">
```
Comment (stevenalexander) [inline alpha/internal-web/app/views/claims-list.html:10]: table is getting fixed width from somewhere

File: alpha/internal-web/app/views/claims-list.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{% extends "layout.html" %} {% block page_title %} Assisted Prison Visit Service (Internal) {% endblock %} {% block content %}
+
+<main id="content" role="main">
+  {% include "includes/phase_banner_alpha.html" %}
+
+  <div class="grid-row">
+
+    <div class="column-two-thirds">
```
Comment (stevenalexander) [inline alpha/internal-web/app/views/claims-list.html]: as above don't need grid layout
https://apvs-alpha-ux.herokuapp.com/examples/elements/grid-layout

**Pull request merged**